data/account/participationRegistry.go

```diff
@@ -285,6 +286,12 @@ func makeParticipationRegistry(accessor db.Pair, log logging.Logger) (*participa
 	err := db.Initialize(accessor.Wdb, migrations)
 	if err != nil {
 		accessor.Close()
+
+		var sqlError *sqlite3.Error
+		if errors.As(err, &sqlError) {
+			return nil, fmt.Errorf("unable to initialize participation registry database: %w.  Sql error - Code: %d, Extended Code: %d", err, sqlError.Code, sqlError.ExtendedCode)
+		}
```

Comment:
Could you put this handling in `db.Initialize`? That way if other people adopt the helper functions there, they will all benefit from this.